utils/fscrypt/Makefile

```diff
@@ -0,0 +1,57 @@
+include $(TOPDIR)/rules.mk
```

Comment:
Does this Makefile need a license, similar to other package Makefiles in this repo? (SPDX line)

Reply:
Not sure, seems like a mixed-bag in the Makefile here